images/ci-prune-buildcache/ci_buildcache_prune.py

```diff
@@ -0,0 +1,490 @@
+import argparse
+import boto3
+from collections import defaultdict
+from contextlib import contextmanager
+from datetime import datetime, timedelta, timezone
+import gitlab
+import multiprocessing as mp
+import multiprocessing.pool as pool
+import os
+from io import BytesIO
+from urllib.parse import urlparse
+
+import buildcache
+import helper
+from pruner import DirectPruner, IndexPruner, OrphanPruner
+
+# Script Config
```

Comment:
We should configure sentry here as well, so we get notified of any errors.

Reply:
Fixed in https://github.com/spack/spack-infrastructure/pull/415/commits/74a0696e3dc472041945f5a9162b54969ed7b004